tests/helm/terraform/static/all_fields.output.yaml

```diff
@@ -641,7 +641,7 @@ data:
 
             terraform import \
                 -var="create_fields=1" \
-                sumologic_field."${FIELD}" "${FIELD_ID}"
+                sumologic_field."${FIELD}"[0] "${FIELD_ID}"
```

Comment:
why do we need the first index?

Reply:
Because this is a resource configured with count: https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/ca05b7ff08182b781193e295955e2372d571856e/deploy/helm/sumologic/conf/setup/fields.tf#L3, and this syntax is how you're supposed to import them.

Reply:
Nice find :clap: